README.md

```diff
@@ -1,33 +1,12 @@
 <!---
-start of trulens_eval/gh_top_intro.md
-NOTE: This content is from trulens_eval/gh_top_intro.md and is merged into
-README.md . If you are editing README.md, your changes will be overwritten.
+start of docs/trulens_eval/intro.md
+NOTE: This content is from docs/trulens_eval/intro.md and is merged into
+trulens_eval/README.md . If you are editing README.md, your changes will be overwritten.
 -->
-
-![PyPI - Version](https://img.shields.io/pypi/v/trulens_eval?label=trulens_eval&link=https%3A%2F%2Fpypi.org%2Fproject%2Ftrulens-eval%2F)
-![Azure DevOps builds (job)](https://img.shields.io/azure-devops/build/truera/5a27f3d2-132d-40fc-9b0c-81abd1182f41/9)
-![GitHub](https://img.shields.io/github/license/truera/trulens)
-![PyPI - Downloads](https://img.shields.io/pypi/dm/trulens_eval)
-[![Slack](https://img.shields.io/badge/slack-join-green?logo=slack)](https://communityinviter.com/apps/aiqualityforum/josh)
-[![Docs](https://img.shields.io/badge/docs-trulens.org-blue)](https://www.trulens.org/trulens_eval/getting_started/)
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/truera/trulens/blob/releases/rc-trulens-eval-0.17.0/trulens_eval/examples/quickstart/colab/langchain_quickstart_colab.ipynb)
```

Comment:
Why delete the shield icons?

Reply:
Oh might be from overriding the root readme with the trulens_eval one. Can keep those